quiz/docker-compose.prod.yaml

```diff
@@ -14,11 +14,49 @@ services:
     restart: on-failure
     networks:
       - proxy-web
+      - backend_network
     depends_on:
       - web
+      - backend_quiz
     ports:
       - "80:80"
+    volumes:
+      - static_volume:/app/staticfiles
+
+  backend_quiz:
+    build:
+      context: ./backend
+      dockerfile: Dockerfile.prod
+    env_file:
+      - ./backend/.envs/.env
+      - envs/.db.env
+    command: gunicorn config.wsgi:application --bind 0.0.0.0:8000
+    restart: on-failure
+    depends_on:
+      - backend_db
+    networks:
+      - backend_network
+      - backend_db_network
+    volumes:
+      - static_volume:/app/staticfiles
+
+  backend_db:
```

Comment:
nazwa serwisu backend_db, dla bazy może nie wystarczyć, gdyż będziemy mieć ich wiele, i w wielu projektach wiele, żeby się nie nakładały na siebie może przyjemiemy konwencje, że service do bazy danych nosi nazwe servicu, dla którego baza jest potrzebna plus db?

Reply:
czyli w tym wypadku backend_quiz_db ?

Reply:
a gdyby wprowadzić:
quiz_backend
quiz_postgres
quiz_web
quiz_mobile
quiz_redis

?

Reply:
No fakt, wygląda to bardziej elegancko i komendy łatwiej wpisywać.

Reply:
Zmieniłam, sprawdziłam czy dev i prod działają. Nie zmieniłam tylko serwisu proxy, bo jak to zrobiłam to na devie przestał wyświetlać się ekran powitalny Vite React TypeScript i trochę boję się tego teraz dotknąć.

Reply:
już działa wszystko